v1.24.0-rc (#2631)
googleforgames/agones: site/content/en/blog/releases/1.24.0-rc.md

---
title: "1.24.0 - Release Candidate"
linkTitle: "1.24.0-rc"
date: "2022-06-15"
---
This is the 1.24.0-rc Release of Agones.

> This release has lots of important bug fixes, documentaton. This includes:

Reviewer: ```suggestion
This release has lots of important bug fixes and documentation. This includes:
```

> * Fix for cross-platform build for simple-game-server.

Reviewer: ```suggestion
* ARM64 image for the simple-game-server example.
```

> * Scale fleet if a game server is allocated from an older version.

Reviewer: ```suggestion
* Fix for scaling a fleet when a game server is allocated from an older version.
```

* Fix for sdkServer.logLevel nil in Gameserver in local-mode.

> * sdkServer.logLevel in Gameserver config doesn't seem to be used in local mode.

Reviewer: ```suggestion
* sdkServer.logLevel in Gameserver config can now be used in local mode.
```

> * ARM64 agones-controller image.

Reviewer: I'd like us to highlight ar a bit more, so howabout something like:

```suggestion

We also believe we've fixed all the issues with running on ARM64 clusters with this release, so please give a try (especially while we are in RC and can potentially fix things before stable next week).
```
